src/components/MultiGestureCanvas/useTapGestures.ts

```diff
@@ -107,6 +120,14 @@ const useTapGestures = ({
     );
 
     const doubleTapGesture = Gesture.Tap()
+        // eslint-disable-next-line @typescript-eslint/naming-convention
+        .onTouchesDown((_evt, state) => {
```

Comment:
Is this disabling the leadingUnderscore rule or something else? If so, why do we need to disable it and not changing the variable name?

Reply:
Yes. Renaming this to `evt` for example will cause TypeScript to throw an error, because the variable is never used. Prefixing an unused function parameter with `_` is a best practice, therefore i disabled the ESLint rule for this line

Reply:
Also, prefixing an unused variable with `_` will prevent TS from throwing this error.

We can either ignore the TS or the ESLint error. We can't omit the parameter completely, because we need the second one (`state`)

Reply:
I think a common pattern I've seen is to use `(_, state)`, which should be fine to use here. If you still need to disable eslint for this line, please add a comment for _why_ it's being disabled (eg. "The first argument is ignored")

Reply:
@tgolen event single `_` will throw the error... Maybe it makes sense to slighly adjust the eslint config and allow `_` prefix? https://typescript-eslint.io/rules/naming-convention/#leadingunderscore--trailingunderscore

<img width="869" alt="image" src="https://github.com/Expensify/App/assets/22820318/02757961-482e-46eb-9a0f-2394ada4cf63">

I see many places in the code where we add `_` prefix for unused variables and then add esling ignore statements 🤔 For example:

<img width="789" alt="image" src="https://github.com/Expensify/App/assets/22820318/dd077d58-741c-4cff-bb75-264a47813e87">

Reply:
I'm fine with continuing to use `_` or `_e` here and just add a `next-line` ignore statement with an additional comment above it like "The first argument is unused, but necessary to have defined". However, it occurs to me that we should perhaps have a better convention for this without changing the eslint rule. What do you think about updating all our code to have `UNUSED` for the name of the param? I think that will pass the linter and also make its purpose super clear.

Reply:
Bump on this convo.

Reply:
@tgolen the issue was resolved. The new changes are hidden, because they added above the line that you used for you comment, so changes are not visible on "conversation" tab, if you open  "Files changed" tab you'll see a full difference and will see a new comment.